index.js

```diff
@@ -39,9 +39,12 @@ module.exports = function(content) {
 	}
 	var root = config.root;
 	var links = attrParse(content, function(tag, attr) {
-		var item = tag + ":" + attr;
 		var res = attributes.find(function(a) {
-		  return item.indexOf(a) >= 0;
+			if (a.startsWith(':')) {
```

Comment:
Well, just checked on [node.green](http://node.green/#ES2015-built-in-extensions-String-prototype-methods-String-prototype-startsWith). Supported on Node >= `v4` (the `v0.12` requires the `--harmony` flag... 😅   No worries, in this case `a.charAt(0) === ':'` is exactly the same so changing it right now. 😉